script/construction_drone.lua

```diff
@@ -1928,11 +1928,12 @@ process_drone_command = function(drone_data, result)
         return
     end
 
-    if drone_data.player and drone_data.player.valid and drone_data.player.character then
-        drone.speed = max(drone_data.player.character_running_speed * 1.2, 0.2)
-    else
-        drone.speed = 1
-    end
+    -- todo: what does this do??
+    -- if drone_data.player and drone_data.player.valid and drone_data.player.character then
+    --     drone.speed = max(drone_data.player.character_running_speed * 1.2, 0.2)
+    -- else
+    -- end
+    drone.speed = 0.2
```

Comment:
Have you tested this speed? Does it seem reasonable, also if we equip eg. power armor with exoskeleton, or travel in car or tank?

Reply:
hi, i am currently using it in my play-through, i am only on foot and in my car. it seems slower than the car but just walking is enough for them to catch up easily.
exoskeleton i have not tested yet